### PR TITLE
Manage comments with useFieldArray

### PR DESCRIPTION
**Pull request by fabian202**

### CODE REVIEW

File: src/components/pages/ProgramDetailsPage/ProgramStatus/index.js [hunk 1]
```diff
@@ -372,37 +382,38 @@ const ProgramStatus = (
          <Col md={2}>
           <Label className="mb-0">Assigned:</Label>
           </Col>
-          <Col md={6}>
-            <StyledModeSelector isUnassigned={statusName === STATUS_VERIFIED}>
+          <Col md={4}>
+            {console.log(`ss`, program, isEmpty(program?.assignment?.assignedUser))}
```
Comment (cesar82) [inline src/components/pages/ProgramDetailsPage/ProgramStatus/index.js:386]: uppsss, remove it

File: src/components/molecules/CommentList/index.js [hunk 2]
```diff
@@ -1,57 +1,113 @@
...
+            {...{
+              control,
+              register,
+              index,
+              errors,
+              watch,
+              setValue,
+              setCanComment,
+            }}
+            onDelete={(id) => handleDelete(id)}
```
Comment (normandanielmq) [inline src/components/molecules/CommentList/index.js:104]: Would it work with `onDelete={handleDelete}` ?

File: src/components/molecules/CommentList/index.js [hunk 3]
```diff
@@ -1,57 +1,113 @@
...
+            name={commentName}
+            key={comment.id}
+            onCancel={(ix) => remove(ix)}
```
Comment (normandanielmq) [inline src/components/molecules/CommentList/index.js:107]: `onCancel={remove}` ?

File: src/components/pages/ProgramDetailsPage/ProgramStatus/index.js [hunk 4]
```diff
@@ -61,6 +62,7 @@ const ProgramStatus = (
   const key = 'Program Status'
   const noLongerAvailableCheckbox = useRef()
   const notActiveCheckbox = useRef()
+  const reCheckWatch = watch('')
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/ProgramStatus/index.js:65]: Is this doing something?

**Review by normandanielmq — APPROVED** [review]

File: src/components/pages/ProgramDetailsPage/ProgramStatus/index.js (hunk 4 above)
Comment (fabian202) [reply src/components/pages/ProgramDetailsPage/ProgramStatus/index.js:65]: I think it can be removed, can you confirm @cesar82

**Review by cesar82 — APPROVED** [review]

**Pull request merged**